app/controllers/three_d_secure_controller.js

```diff
@@ -103,6 +104,15 @@ module.exports = {
         data.md = md
       }
       responseRouter.response(req, res, views.AUTH_3DS_REQUIRED_OUT_VIEW, data)
+    } else if (worldpayChallengeJwt) {
+      const challengeUrl = charge.gatewayAccount.type === 'live'
+        ? process.env.WORLDPAY_3DS_FLEX_CHALLENGE_LIVE_URL
+        : process.env.WORLDPAY_3DS_FLEX_CHALLENGE_TEST_URL
+      let data = {
+        issuerUrl: challengeUrl,
```

Comment:
Maybe `issuerUrl` should be renamed as it’s no longer necessarily a URL provided by the card issuer.